packages/tokens/src/global/elevations.ios.ts

```diff
@@ -0,0 +1,21 @@
+import { elevate } from '@gympass/yoga-common';
+import colors from './colors';
+
+const elevateOptions = {
+  color: colors.medium,
+  depth: 1,
+  spread: false,
+  fallback: true,
+};
+
+const [zero, small, medium, large] =
+  elevate(elevateOptions) as [string, string, string, string];
```

Comment:
**question**: `as` casts hide potential errors (it's us saying we know better than typescript). Do we really need this?